src/spatial_ref/srs.rs

```diff
@@ -7,6 +7,146 @@ use std::str::FromStr;
+                south_latitude_deg,
+                east_longitude_deg,
+                north_latitude_deg,
+            )
+        };
+        if ret_val == 0 {
+            return Err(GdalError::OctError {
+                method_name: "OCTCoordinateTransformationOptionsSetAreaOfInterest",
+            });
+        }
+        Ok(())
+    }
+
+    /// Sets the desired accuracy for coordinate operations.
+    ///
+    /// Only coordinate operations that offer an accuracy of at least the one specified will be
+    /// considered.
+    ///
+    /// An accuracy of 0 is valid and means a coordinate operation made only of one or several
+    /// conversions (map projections, unit conversion, etc.) Operations involving ballpark
+    /// transformations have a unknown accuracy, and will be filtered out by any dfAccuracy >= 0
+    /// value.
+    ///
+    /// If this option is specified with PROJ < 8, the OGR_CT_OP_SELECTION configuration option
+    /// will default to BEST_ACCURACY.
+    #[cfg(all(major_ge_3, minor_ge_3))]
+    pub fn desired_accuracy(&mut self, accuracy: f64) -> Result<()> {
+        let ret_val = unsafe {
+            gdal_sys::OCTCoordinateTransformationOptionsSetDesiredAccuracy(self.inner, accuracy)
+        };
+        if ret_val == 0 {
+            return Err(GdalError::OctError {
+                method_name: "OCTCoordinateTransformationOptionsSetDesiredAccuracy",
+            });
+        }
+        Ok(())
+    }
+
+    /// Sets whether ballpark transformations are allowed.
+    ///
+    /// By default, PROJ may generate "ballpark transformations" (see
+    /// <https://proj.org/glossary.html>) when precise datum transformations are missing. For high
+    /// accuracy use cases, such transformations might not be allowed.
+    ///
+    /// If this option is specified with PROJ < 8, the OGR_CT_OP_SELECTION configuration option
+    /// will default to BEST_ACCURACY.
+    #[cfg(all(major_ge_3, minor_ge_3))]
+    pub fn set_ballpark_allowed(&mut self, ballpark_allowed: bool) -> Result<()> {
+        let ret_val = unsafe {
+            gdal_sys::OCTCoordinateTransformationOptionsSetBallparkAllowed(
+                self.inner,
+                ballpark_allowed as libc::c_int,
+            )
+        };
+        if ret_val == 0 {
+            return Err(GdalError::OctError {
+                method_name: "OCTCoordinateTransformationOptionsSetBallparkAllowed",
+            });
+        }
+        Ok(())
+    }
+
+    /// Sets a coordinate operation.
+    ///
+    /// This is a user override to be used instead of the normally computed pipeline.
+    ///
+    /// The pipeline must take into account the axis order of the source and target SRS.
+    ///
+    /// The pipeline may be provided as a PROJ string (single step operation or multiple step
+    /// string starting with +proj=pipeline), a WKT2 string describing a CoordinateOperation, or a
+    /// "urn:ogc:def:coordinateOperation:EPSG::XXXX" URN.
```

Comment:
Is there a canonical description on these operation definitions you could link to?

Reply:
I have absolutely no idea what any of this even means, I just copied the docs from upstream GDAL.

Reply:
I found https://gdal.org/tutorials/osr_api_tut.html#advanced-coordinate-transformation which sheds some light. I don't think our API docs are the right place to expand this fully, so I added a link.